### PR TITLE
Added Dto classes for the entities

### PR DESCRIPTION
**Pull request by b-laci**

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
@b-laci Ivan ti je GOTO guy za svu pomoc i pull requestove, ja samo nadgledam, kad stignem. :)

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.Setter;
+
+import java.util.Date;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
+@Getter
+@Setter
+public class PomodoroDto {
+
```
Comment (ksajvan) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: Why did you rename Dto class? Since it already resides in dto package, there is no need to have Dto explicitly in class name.

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/TeamDto.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
+@Getter
+@Setter
+public class TeamDto {
+
```
Comment (ksajvan) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/TeamDto.java]: check comment above about class name

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/UserDto.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
+@Getter
+@Setter
+public class UserDto {
+
```
Comment (ksajvan) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/UserDto.java]: same for this class

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java (hunk 1 above)
Comment (b-laci) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: I named the classes like that just for readability, I think it is easier to make difference between entities and dto classes

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package eu.execom.labs.pomodorospringreact.dto;
+
```
Comment (ksajvan) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: why those files are defined in web module?

File: web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java (hunk 1 above)
Comment (ksajvan) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: in that case, leave them as they were
